### DISCUSSION TITLE
A trial for locating mouse press on an image

### DISCUSSION THREAD

**Discussion by yuvalharr** [jspsych/jsPsych#1140] [Q&A]
Hey there,

I would be really glad to get any help with implementing a trial that I couldn't figure out how to do it myself.

In this trial I want to present an image of a picture and a grid of squares on top of it.
I then want the subjects to press with their mouse on one of the squares (the one in which they spot a target object). 
I would like to know from this press if they pressed on the correct square or not, and continue with the next trials.

Any ideas of how I should approach this?

Here is a picture demonstrating how it should look:

![image](https://user-images.githubusercontent.com/40396020/97110967-c053bd00-16e4-11eb-86cc-7eba1f5f7199.png)

Thanks for any help!

Yuval

**Comment by GEJ1** [comment]
I think you can try using the psychophysycs plugin: https://jspsychophysics.hes.kyushu-u.ac.jp/ made by @kurokida 

Maybe you can try starting with this demo to record the click position: https://www.hes.kyushu-u.ac.jp/~kurokid/jspsychophysics/demos/localize-circle.html

**Comment by becky-gilbert** [comment]
I like @GEJ1's suggestion about using the psychophysics plugin, but just in case that doesn't do what you want, you could also see [this response to a (sort of) similar question](https://github.com/jspsych/jsPsych/discussions/1114#discussioncomment-98515). This comment explains how to edit one of the image plugin files so that it records the mouse click location, and then categorizes the mouse click as being either in the top or bottom half of the image. You could do something similar, but with more x and y ranges so that you can categorize the mouse click in terms of your 4x4 grid squares.
